README.md

```diff
@@ -27,7 +27,35 @@ E.g. this code adds such ability to every Content NodeType:
 
 ![inspector editor](editor-demo.png)
 
-### 2. Render the anchors in the NodeType rendering
+### 2. Make content nodes available in the link search dialog of the backend
+
+Normally, Neos offers only document nodes for selection in its link search dialog. To list the corresponding content 
+nodes as well, you must allow this in the link configuration and set nodeTypes to 
+`['Neos.Neos:Document', 'CodeQ.JumpMarkers:Mixin.SectionConfiguration']`, for example:
+
+```
+'YOUR.Site:Content.Text':
+  superTypes:
+    'Neos.Neos:Content': true
+  ui:
+    label: Text
+    icon: file-text
+    position: 200
+    inlineEditable: true
+  properties:
+    text:
+      options:
+        ui:
+          inline:
+            editorOptions:
+              linking:
+                nodeTypes: ['Neos.Neos:Document', 'CodeQ.JumpMarkers:Mixin.SectionConfiguration']
+```
+
+Note that this allows any content node to be selected as a link target, including those that have not yet been assigned 
+an anchor id and therefore cannot be linked properly in the frontend.
```

Comment:
"including those that have not yet been assigned an anchor id and therefore cannot be linked properly in the frontend."
That's a big weird, I would recommend that we have a fallback to always render ids with node.identifier like this:
```
prototype(CodeQ.JumpMarkers:NodeAnchorId) < prototype(Neos.Fusion:Component) {
    ...

    renderer = Neos.Fusion:Value {
        ...

        fallback = ${q(props.node).property('jumpMarkerTitle') || ('node-' + node.identifier)}
        ...
    }
}

```

Reply:
I am not sure here. Currently a section id or section title has to be actively set by the editor to have some URL with fragment for further use. With this fallback we would switch from opt-in to always-in. Maybe we should add a global switch to turn this feature on and off. Do we really want to show the node id to the outside world?

Reply:
I wrote an aspect handling this, not only nodes with a jumpMarkerTitle or sectionId id are shown.